decidim-conferences/app/views/decidim/conferences/media/_attachments.html.erb

```diff
@@ -0,0 +1,4 @@
+<% if attached_to.attachments.any? %>
+  <%= render partial: "photos", locals: { photos: attached_to.photos } %>
```

Comment:
I'd move these partials into cells

Reply:
Okey to move partial: "photos" into a cell. Because we are overwritting the existing one of Decidim::Core, 
But, move partial "documents" into a cell is more complicated. Because it is used to the whole Decidim, so it may take into account a lot of things out of scope of this PR.

Reply:
👍